### ISSUE TITLE
Title of Archive File used instead of ComicInfo Series Name

### ISSUE THREAD

**Issue by MrMysterius** [Kareadita/Kavita#1631]
**Describe the bug**
Series name defined in `ComicInfo.xml` is not used, instead is the name from the file, which is not correct.

**To Reproduce**
1. Fresh Installation of Kavita.
2. Add manga folder (`manga`)
3. Add subfolder (`Fire Punch`)
4. Add File (`Chapter-1.cb7` `majoa2008#5364` got a copy from me `Mr Mysterius#0020` in DM's on Discord)
5. Scan Library

**Expected behavior**
Using the series name from `ComicInfo.xml` instead of the file name.

**Desktop (please complete the following information):**
- Reading On: Desktop - Windows
- Kavita On: Unraid - Linux Server - Docker
- Browser: Vivaldi

**Screenshots**
![image](https://user-images.githubusercontent.com/25854814/199547612-013854d9-4bfc-4453-833c-d9e94e918ed0.png)
![image](https://user-images.githubusercontent.com/25854814/199547655-53d697fa-df46-4fd1-b8ca-6f8acaebe6a0.png)
![image](https://user-images.githubusercontent.com/25854814/199547704-c180b12a-4a62-47a6-afe7-e0b8665f82bd.png)
![image](https://user-images.githubusercontent.com/25854814/199547783-5f827cd7-c20c-45a4-a140-0c7c55e77feb.png)

`ComicInfo.xml`:
```xml
<?xml version="1.0"?>
    <ComicInfo xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
      <Title>Fire Punch Kapitel 1</Title>
      <Series>Fire Punch</Series>
      <Number>1</Number>
      <Volume>1</Volume>
      <Notes></Notes>
      <Writer></Writer>
      <Genre>Action,Drama,Fantasy,Mystery,Psychological,Romance,SciFi,Shounen,Splatter,Supernatural,Superpower,Violence</Genre>
      <Web></Web>
      <PageCount>64</PageCount>
      <AgeRating>Unknown</AgeRating>
      <Characters></Characters>
      <Pages>
        <Page Image="0" ImageWidth="960" ImageHeight="1508" />
<Page Image="1" ImageWidth="960" ImageHeight="1508" />
<Page Image="2" ImageWidth="960" ImageHeight="1508" />
<Page Image="3" ImageWidth="960" ImageHeight="1508" />
<Page Image="4" ImageWidth="960" ImageHeight="1508" />
<Page Image="5" ImageWidth="960" ImageHeight="1508" />
<Page Image="6" ImageWidth="960" ImageHeight="1508" />
<Page Image="7" ImageWidth="960" ImageHeight="1508" />
<Page Image="8" ImageWidth="960" ImageHeight="1508" />
<Page Image="9" ImageWidth="960" ImageHeight="1508" />
<Page Image="10" ImageWidth="960" ImageHeight="1508" />
<Page Image="11" ImageWidth="960" ImageHeight="1508" />
<Page Image="12" ImageWidth="960" ImageHeight="1508" />
<Page Image="13" ImageWidth="960" ImageHeight="1508" />
<Page Image="14" ImageWidth="960" ImageHeight="1508" />
<Page Image="15" ImageWidth="960" ImageHeight="1508" />
<Page Image="16" ImageWidth="960" ImageHeight="1508" />
<Page Image="17" ImageWidth="960" ImageHeight="1508" />
<Page Image="18" ImageWidth="960" ImageHeight="1508" />
<Page Image="19" ImageWidth="960" ImageHeight="1508" />
<Page Image="20" ImageWidth="960" ImageHeight="1508" />
<Page Image="21" ImageWidth="960" ImageHeight="1508" />
<Page Image="22" ImageWidth="960" ImageHeight="1508" />
<Page Image="23" ImageWidth="960" ImageHeight="1508" />
<Page Image="24" ImageWidth="960" ImageHeight="1508" />
<Page Image="25" ImageWidth="960" ImageHeight="1508" />
<Page Image="26" ImageWidth="960" ImageHeight="1508" />
<Page Image="27" ImageWidth="960" ImageHeight="1508" />
<Page Image="28" ImageWidth="960" ImageHeight="1508" />
<Page Image="29" ImageWidth="960" ImageHeight="1508" />
<Page Image="30" ImageWidth="960" ImageHeight="1508" />
<Page Image="31" ImageWidth="960" ImageHeight="1508" />
<Page Image="32" ImageWidth="960" ImageHeight="1508" />
<Page Image="33" ImageWidth="960" ImageHeight="1508" />
<Page Image="34" ImageWidth="960" ImageHeight="1508" />
<Page Image="35" ImageWidth="960" ImageHeight="1508" />
<Page Image="36" ImageWidth="960" ImageHeight="1508" />
<Page Image="37" ImageWidth="960" ImageHeight="1508" />
<Page Image="38" ImageWidth="960" ImageHeight="1508" />
<Page Image="39" ImageWidth="960" ImageHeight="1508" />
<Page Image="40" ImageWidth="960" ImageHeight="1508" />
<Page Image="41" ImageWidth="960" ImageHeight="1508" />
<Page Image="42" ImageWidth="960" ImageHeight="1508" />
<Page Image="43" ImageWidth="960" ImageHeight="1508" />
<Page Image="44" ImageWidth="960" ImageHeight="1508" />
<Page Image="45" ImageWidth="960" ImageHeight="1508" />
<Page Image="46" ImageWidth="960" ImageHeight="1508" />
<Page Image="47" ImageWidth="960" ImageHeight="1508" />
<Page Image="48" ImageWidth="960" ImageHeight="1508" />
<Page Image="49" ImageWidth="960" ImageHeight="1508" />
<Page Image="50" ImageWidth="960" ImageHeight="1508" />
<Page Image="51" ImageWidth="960" ImageHeight="1508" />
<Page Image="52" ImageWidth="960" ImageHeight="1508" />
<Page Image="53" ImageWidth="960" ImageHeight="1508" />
<Page Image="54" ImageWidth="960" ImageHeight="1508" />
<Page Image="55" ImageWidth="960" ImageHeight="1508" />
<Page Image="56" ImageWidth="960" ImageHeight="1508" />
<Page Image="57" ImageWidth="960" ImageHeight="1508" />
<Page Image="58" ImageWidth="960" ImageHeight="1508" />
<Page Image="59" ImageWidth="960" ImageHeight="1508" />
<Page Image="60" ImageWidth="960" ImageHeight="1508" />
<Page Image="61" ImageWidth="1920" ImageHeight="1508" />
<Page Image="62" ImageWidth="960" ImageHeight="1508" />
<Page Image="63" ImageWidth="960" ImageHeight="1508" />

      </Pages>
    </ComicInfo>
```

**Comment by ThePromidius** [comment]
Adding more context here. Apparently it's just not reading the comicinfo file for some reason. From the logs volume wasn't parsed either and i doubt any other tag did.
Validated the xml just fine so guess it's has to do with something in the scan.

**Comment by majora2007** [comment]
I am looking into this, I will already say this file does not follow the spec for ComicInfo. The ComicInfo.xml is supposed to be at the root of the archive. 

Kavita is lenient and allows fallback, so I will investigate the issue.

**Comment by MrMysterius** [comment]
> I am looking into this, I will already say this file does not follow the spec for ComicInfo. The ComicInfo.xml is supposed to be at the root of the archive.
> 
> Kavita is lenient and allows fallback, so I will investigate the issue.

Follow-up question. Isn't it at the root of my archive? Because this is how one archive (`.cb7`) looks like...

![](https://user-images.githubusercontent.com/25854814/199547704-c180b12a-4a62-47a6-afe7-e0b8665f82bd.png)

**Comment by majora2007** [comment]
No, your archive had a folder inside it, with the images inside that. The folder is at the root of the archive, so should the comicinfo.

This is fixed however in v0.6.1 which is out now.
